### ISSUE TITLE
I feel as if Neos Essentials is not properly Vetted or checked enough.

### ISSUE THREAD

**Issue by InsaneGrox** [Neos-Metaverse/NeosPublic#1844]
## What topic do you want to discuss?
Neos Essentials, Specifically how content is vetted/handled inside of Neos Essentials, specifically the problems with the way it is currently handled, such as items being outdated and broken, items being there despite licensing not allowing it, and other stuff.

## Goal of discussion
To hopefully find a solution to these problems, personally I'd just have shifty or someone else spend a day working on neos essentials, and then for the future have somebody pay closer attention to what gets added, as well as make sure that who has access to adding stuff is trustworthy with that stuff.

## Discussion:
This is mostly related to the Da-vali fiasco from about a month ago, specifically the core of why that happened, as well as other concerns I have, I feel like Neos Essentials isn't being vetted properly, to where content is being added without much thought, including assets that break license agreements, one potential example that I'm not 100% sure about but I do question is the inclusion of a skybox that is a render from one of the batman arkham games (arkham city specifically), and the legality of that, (the most prime example is the already resolved inclusion of the DaVali, which has since been removed.),the other issue is content from a while ago never being updated when they break, a perfect example of this would be nearly every weapon in the gadgets folder having incorrect tool offsets, another problem I feel is that a lot of what is in neos essentials isn't being set up properly, avatars in particular almost never have anything assigned into the eye manager for eye tracking, or the expression driver for lip tracking, despite having the shape keys available, prime examples being the nargacouga (has eye widen shape keys that are unassigned), and the Neos Expression Bot, which have several eye tracking shape keys that are completely unused by the eye manager, I also feel that the full body tracking calibration on these avatars is also incredibly off, sometimes being inconsistant, the worst offender being the yeen.
I also feel that there are things that could be in neos essentials that aren't in there, perfect example being 3d models from https://3dmodelhaven.com/, skyboxes from https://hdrihaven.com/, and materials from https://texturehaven.com/, all being licensed as CC0, and being incredibly high quality assets, even beating CC0textures assets in my opinion.
TLDR: Someone should probably go through neos essentials and fix some things, also someone's allowed to add stuff to it who really shouldn't be allowed.

**Comment by ProbablePrime** [comment]
The same in a lesser extent goes for EssentialTools. 

I feel its turned into just "Tools" there are hundreds of tools in there. They're cool, but im not sure if they qualify as essential. 

I made a folder called ActuallyEssentialTools which I now use in replacement of EssentialTools for quick locating of the tools I need for tutorials. This was mostly a meme/joke but I've seen other users enjoying it too.

**Comment by Frooxius** [comment]
The main problem is that those two folders have kinda outgrown what they were made for, so they become a bit messy over time and with the inclusion of various community made/setup items too. They're still more of temporary placeholders before the workshop, which will completely replace those.

We do go over it time from time to clean it up and remove any problematic items and update the ones we have, but it takes some time away from other tasks, so some stuff will be put through. If you find any existing items that shouldn't be there, please let us know asap and we'll get rid of them (or the community folder they're in).

For some fancy stuff like extra blendshapes and assignments on the avatar, those are more cosmetic. We'd like to set those over time, but again, it's a time consuming task and we feel it's better for community workshop, where the workload can be more spread out.

We do have CC0 textures right now at least. I'd like to include other resources as well like the ones you mentioned, but I need to write new scripts to import those (and fix the CC0 Textures one since they changed their structuring). I'm kinda waiting on the workshop for that as well, since that's going to be much better place for those anyway.

**Comment by Konshu** [comment]
Would it be best to have one of the community managers and possibly some of the builders in here go through it as a review team and gut them? I am not sure whose tasks would be impacted by updating this but the community is pretty effective at this kind of thing and likely could see a fast turnaround. Pretty sure this is the main reason the community created Starter Pack was generated to work around similar issues.

**Comment by ProbablePrime** [comment]
I noticed the "Grebbable Dispenser" in Neos Essentials -> Gadgets. I don't think it should be there. While a great gadget, it can cause problems for newer users.
- It causes motion sickness in those new to VR
- Users might not be aware what it does leading to a loss of control/confusion
- Other users can use it maliciously by tricking unaware users from being trapped

I was under the impression, that the Neos Essentials folder was being updated in a few ways to remove some potentially problematic gadgets from it. Which is why I'm bringing this up.

**Comment by ProbablePrime** [comment]
Additionally, It feels like The Starter Pack is misplaced. A folder was created called "Community Public Folders" Which currently only contains Creator Jam's Folder.

It feels like the starter pack should be in Community Public Folders too. 

Please, don't get me wrong. The Starter Pack is cool. Its placement is just confusing.

**Comment by InsaneGrox** [comment]
The reverse natavali avatars both do not have hand posers... the fingers as a result are completely static despite being rigged...
(also I'm fairly certain those shouldn't even be in there considering those are part of the LOF discord server... just like the da'vali is part of the AA discord server... should probably get rid of them if permission wasn't gained)
This is exactly what I mean by I feel like nothing in Neos Essentials is vetted properly...

**Comment by Electronus** [comment]
@InsaneGrox  I talked with Aegis and the reverse Natvali is fine to be in there, they have explicit permission.

**Comment by InsaneGrox** [comment]
> @InsaneGrox I talked with Aegis and the reverse Natvali is fine to be in there, they have explicit permission.

good, just hoping this isn't another da'vali incident waiting to happen... (still the fingers should probably be fixed)
